src/app/core/services/permission.service.ts

```diff
@@ -0,0 +1,47 @@
+import { Injectable } from '@angular/core';
+import {NetService} from "./net.service";
+import {Permission} from "../model/permissions";
+import {Observable} from "rxjs";
+import urlJoin from "url-join";
+import {map} from "rxjs/operators";
+
+@Injectable({
+  providedIn: 'root'
+})
+export class PermissionService {
+
+  constructor(private netService: NetService) {
+  }
+
+  /**
+   * Sets the given permissions for the given user.
+   * @param userId Id of the user the permissions shall be set for.
+   * @param permissionsToSet
+   */
+  setPermissionsFor(userId: string, permissionsToSet: Permission[]): Observable<void> {
+    return this.netService.putJSON(urlJoin('/permissions', 'user', userId), permissionsToSet, {});
+  }
+
+  /**
+   * Retrieve permissions of the given user.
+   * @param userId Id of the user to retrieve permissions for.
+   */
+  getPermissions(userId: string):Observable<Permission[]>{
```

Comment:
Permissions from what/who?
```suggestion
  getPermissionsByUser(userId: string):Observable<Permission[]>{
```